packages/framework-provider-kubernetes/src/library/graphql-adapter.ts

```diff
@@ -31,5 +31,5 @@ export const handleResult = async (
     ...headers,
   },
   statusCode: 200,
-  body: result ? JSON.stringify(result) : '',
+  body: result ? result : undefined,
```

Comment:
If you're returning undefined you can avoid this:
```suggestion
  body: result,
```

Reply:
I though it was null if you don't use undefined.